project-2-e-commerce/src/component/confirm-page/index.js

```diff
@@ -0,0 +1,96 @@
+                                <div className='fs-default mt-1'>{`${district}, ${city}`}</div>
+                            </div>
+                            <div className='confirm-info-item w-50 pr-1  mt-1'>
+                                <h3 className='fs-2'>{t('payment methods')}</h3>
+                                <div className='fs-default mt-1'>{t('payment on delivery')}</div>
+                            </div>
+                            <div className='confirm-info-item w-50 pl-1  mt-1'>
+                                <h3 className='fs-2'>{t('trans methods')}</h3>
+                                <div className='fs-default mt-1'>{t('deliver to home')}</div>
+                            </div>
+                        </div>
+                    </div>
+                    <div className="confirm-orders lg-6 sm-12">
+                        <div className='bg-white b-green h-100'>
+                            <div className='pay-order-heading bb-default p-2'>
+                                <h3 className='fs-2'>{code}</h3>
+                            </div>
+                            <div className='pay-order-body p-2'>
+                                <div className='pay-cart-list bb-default pt-1 pb-1'>
+                                    {list && list.map(item => <CartItem {...item} key={item.id}/>)}
+                                </div>
+                                <div className='pay-calculate pt-1 pb-1 bb-default'>
+                                    <div className='pay-calculate-item mb-1 d-flex jc-space-btw fs-default'>
+                                        <span>{t('temporary')}</span>
+                                        <span>{formatPrice(total, t)}</span>
+                                    </div>
+                                    <div className='pay-calculate-item d-flex jc-space-btw fs-default'>
+                                        <span>{t('transportation fee')}</span>
+                                        <span>{formatPrice(transFee, t)}</span>
+                                    </div>
+                                </div>
+                                <div className='pay-total pt-1 pb-1 d-flex jc-space-btw fs-default'>
+                                    <span className='fs-2'><strong>{t('total')}</strong></span>
+                                    <span className='text-blue fs-2'>{formatPrice(totalOrders, t)}</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className='confirm-option flex-center mt-3'>
+                    <button className='back-to-products pay-order-btn fs-2 text-white pl-2 pr-2 pt-1 pb-1'>
+                        {t('back to products page')}
+                    </button>
+                    {isLogged ? <Link className="text-green fs-default ml-2">{t('go to orders page')} &#62;</Link> : null}
```

Comment:
```suggestion
                    {isLogged ? <Link to='/...' className="text-green fs-default ml-2">{t('go to orders page')} &#62;</Link> : <Link to='/login-page'>Login now</Link>}
```
Nếu người dùng chưa đăng nhập thì hiện button login thay vì null nhé